ERROR:root:Exception in main()
Traceback (most recent call last):
  File "/home/manjos/PycharmProjects/NN/Keras Resnet.py", line 39, in <module>
    exit(main())
NameError: name 'main' is not defined
